package.json

```diff
@@ -28,7 +28,7 @@
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "release": "multi-semantic-release --ignore-packages=packages/**",
     "test": "turbo run test --filter=!./appeals/*",
-    "tscheck": "turbo run prisma-generate && turbo run tscheck --parallel --filter=!@pins/api --filter=!@pins/appeals.api",
+    "tscheck": "npm run prisma-generate --workspace=@pins/api && npm run prisma-generate --workspace=@pins/appeals.api  && turbo run tscheck --parallel --filter=!@pins/api --filter=!@pins/appeals.api",
```

Comment:
I think npm run test will also run prisma-generate, and these are running sequentially (looks like we're still filtering on this though, so not sure if this helps)

Reply:
Yeah good point. Weirdly `turbo` has a `--parallel` flag, but even without that prisma-generate appeared to run in parallel from the logs!

Reply:
`tscheck` seems to fail often, so that's why I targeted that first